bin/clover/src/spec/props.ts

```diff
@@ -289,7 +289,7 @@ export function createScalarProp(
     validationFormat: null,
     defaultValue: null,
     funcUniqueId: null,
-    inputs: null,
+    inputs: [],
```

Comment:
I wonder if this can break something on import, making all props think they have values set by functions. Did you test for that?

Reply:
Not having this breaks setting region and aws credential as we don't set the input on the props at all if the list is null. Imported assets seem good